infrastructure/cdn-in-a-box/traffic_vault/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN rm -rfv /etc/riak/prestart.d/* /etc/riak/poststart.d/*
 RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
  && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends \
  && apt-get update \
- && DEBIAN_FRONTEND=noninteractive apt-get install -y vim.tiny wget net-tools sudo net-tools ca-certificates unzip dnsutils \
+ && DEBIAN_FRONTEND=noninteractive apt-get install -y vim.tiny wget net-tools sudo net-tools ca-certificates unzip dnsutils gettext-base \
```

Comment:
I know this isn't something you did, but as long as we're touching this line, can you remove `vim.tiny`, `wget`, `sudo` and the extra `net-tools`? That container has no real use for those, and it's making the thing bigger than it needs to be.

Reply:
ok, I'll remove them